requirements.txt

```diff
@@ -1,2 +1,3 @@
 apluslms-yamlidator
 appdirs >=1.4.0, <2
+colorama >=0.4.1
```

Comment:
`colorama ~= 0.4.1` would be preferred (appdirs should be updated to use that some day..).